src/traces/formatTraces.test.ts

```diff
@@ -95,4 +97,131 @@ describe('FormatTraces', () => {
       });
     });
   });
+  describe('createTraceDataFrame', () => {
+    it('should return in the data frame the fields needed for trace view', () => {
+      const targets = [{ refId: 'A' }];
+      const traceDataFrameResult = createTraceDataFrame(targets, spanListResponse as OpenSearchSpan[]);
+      expect(traceDataFrameResult.data.length).toEqual(1);
+      expect(traceDataFrameResult.key).toEqual('A');
+      const singleDataFrame = traceDataFrameResult.data[0];
+      expect(singleDataFrame.meta.preferredVisualisationType).toEqual('trace');
+      expect(singleDataFrame.fields.find(field => field.name === 'traceID').values).toEqual(
+        new ArrayVector(['0000000000000000213ce26adf2b30d0', '0000000000000000213ce26adf2b30d0'])
+      );
+      expect(singleDataFrame.fields.find(field => field.name === 'duration').values).toEqual(
+        new ArrayVector([227061000 * 0.000001, 525654000 * 0.000001])
+      );
+      expect(singleDataFrame.fields.find(field => field.name === 'serviceName').values).toEqual(
+        new ArrayVector(['driver', 'mysql'])
+      );
+      expect(singleDataFrame.fields.find(field => field.name === 'parentSpanID').values).toEqual(
+        new ArrayVector(['00ce90d301af791e', '008cf73f4305cbf6'])
+      );
+      expect(singleDataFrame.fields.find(field => field.name === 'spanID').values).toEqual(
+        new ArrayVector(['3a5ea3d834fc316d', '153f525e711e84a8'])
+      );
+      expect(singleDataFrame.fields.find(field => field.name === 'operationName').values).toEqual(
+        new ArrayVector(['/driver.DriverService/FindNearest', 'SQL SELECT'])
+      );
+      expect(singleDataFrame.fields.find(field => field.name === 'startTime').values).toEqual(
+        new ArrayVector([
+          new Date('2023-04-11T11:14:31.243838Z').getTime(),
+          new Date('2023-04-11T11:14:30.717151Z').getTime(),
+        ])
+      );
+      // cSpell:disable
```

Comment:
maybe better to add the list of weird words rather than not use cspell? idk I guess either way haha

Reply:
I disabled this block cause these words are probably just a one-time thing specific to this test and they most likely won't be repeated anywhere else